front-end/src/app/reports/f3x/create-workflow/create-f3x-step1.component.ts

```diff
@@ -235,22 +236,40 @@ export class CreateF3XStep1Component implements OnInit, OnDestroy {
       summary.form_type = F3xFormTypes.F3XT;
     }
 
-    this.f3xSummaryService.create(summary, this.formProperties).subscribe((report: F3xSummary) => {
-      if (jump === 'continue') {
-        // Save report to Cash On Hand in the store if necessary by pulling the reports table data.
-        this.reportService.getTableData().subscribe();
+    //Observables are *defined* here ahead of their execution
+    const create = this.f3xSummaryService.create(summary, this.formProperties);
+    // Save report to Cash On Hand in the store if necessary by pulling the reports table data.
+    const tableData = this.reportService.getTableData();
```

Comment:
I think this will need a pipe(takeUntil(this.destroy$)

Reply:
:+1:

Reply:
no need for takeUntil.  There is one subscription in this function and it has a takeUntil at like 255